src/engraving/rendering/iscorerendering.h

```diff
@@ -84,17 +85,40 @@ class TimeSig;
 }
 
 namespace mu::engraving::rendering {
-class IScoreRendering : MODULE_INTERNAL_INTERFACE
+class IScoreRendering : MODULE_EXPORT_INTERFACE
 {
-    INTERFACE_ID(IEngravingRendering)
+    INTERFACE_ID(IScoreRendering)
 
 public:
     virtual ~IScoreRendering() = default;
 
-    // Layout Score
-    virtual void layoutRange(Score* score, const Fraction& st, const Fraction& et) = 0;
+    // Main interface
+
+    virtual void layoutScore(Score* score, const Fraction& st, const Fraction& et) const = 0;
 
-    // Layout Item
+    struct Options
```

Comment:
Should we rename this to `DrawOptions` or `PaintOptions` to distinguish it from layout options?